<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookSearch</class>
 <widget class="QWidget" name="BookSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>551</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入书名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bookname">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>结果如下：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BorrowBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>借书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入作者名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_author">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
